hazelcast/src/main/java/com/hazelcast/map/LocalMapStats.java

```diff
@@ -274,4 +274,11 @@
      */
     Map<String, LocalIndexStats> getIndexStats();
 
+    // todo reconsider exposure of replication stats, probably better encapsulated in separate "LocalReplicationStats"?
+    // these stats reflect full/diff partition replication stats with local map partitions as the migration source
+    long getDifferentialPartitionReplicationCount();
+    long getFullPartitionReplicationCount();
+    long getDifferentialReplicationRecordCount();
+    long getFullReplicationRecordCount();
```

Comment:
Having these methods in `LocalReplicationStats` seems better to me also.(please also add methid javaDocs) 
Another thing, does it make sense to put these new fields in `LocalRecordStoreStats`? It is replicated to new destination during migrations.